packages/server/lib/cloud/api/index.ts

```diff
@@ -227,6 +232,16 @@ const isRetriableError = (err) => {
     (err.statusCode == null)
 }
 
+function noproxyPreflightTimeout (): number {
```

Comment:
is noproxy one word? I'm not sure if this is cased correctly or not 😂 
```suggestion
function noProxyPreflightTimeout (): number {
```

Reply:
Eh, it's likely temporary until we re-assess axios.